MyCryptoHQ/MyCrypto
# Update Styling and Variables (Mostly dashboard)

### Things That Changed

1. New variables, lots of finding and replacing
2. Changed order of the ActionTiles on dashboard after talking with Jordan (send, swap, request, buy, hw)
3. Made the "add account" button look the same and be styled the same across Account panel and Address book panel
4. Added some tooltips to settings; translated some random strings that I found
5. **Dashboard spacing updates.** This will now allow the two top columns lower most item to stretch-to-fill so that their bottoms line up, regardless of if the token panel has a lot of tokens, none, or account panel has a lot of accounts or none. The bulk of the work done in this commit (57b6647) was about getting the margins and paddings to be consistent across the dashboard and settings pages. I suspect there may be some weirdness on mobile or tablet sizes, but I'll fix those later as they were already funky anyways. Not a huge deal.
6. Slightly modified **layout for the token details stuff** and fixed the weird padding / scrollbar that was happening.
7. In the token details panel, I added padding around the clickable areas so that the clickable area would be bigger. This should aid with some of the really small SVGs as they essentially have an invisible clickable area around them now. This increased it from ~20px->~50px (~50px is considered accessible)

### Steps to Test

**UI**
1. Look at the dashboard. Resize the screen. Note anything that looks funky
2. Scan for tokens. Note anything that looks funky.
3. Go to settings. Note anything that looks funky.

**UX**
1. Take a look at a couple token details. It should be easier to read and consume the info, and be less tall.
2. Give feedback on new order of the action tiles (top left 5 buttons on the dashboard) (send, swap, request, buy, hw)

**Technical**
1. If it runs, it should be good. This really was mostly finding and replacing strings with variables and CSS work.
2. Note any errors or warnings. 
3. Test yarn start and yarn build (or whatever it is)
4. Teach taylor whatever she needs to know about linting errors because I'm still force pushing (i'm so sorry)



### Note to future frustrated person
- If something is wonky from this PR, check `57b66479a31d144940e8a8957b8007d7811281b6` and `60968238ab4c9b3be7a4981694a6b26ccb0fc62b` and `22089c291f5beb31ef9cc06d941ed50e41674a6e` first as the issue is far more likely to be in one of those than the others.
- I tried to split the commits up so that if we have to blame something or revert something, it's okay.
- I'm so sorry. I just wanted to make variables. I didn't mean to cause you such pain. 😭


### Other Notes (posted in Slack originally)

I fixed some weird padding and now you have motherfucking variables because, honestly, how do you even live. 😂

Here's the scale:
XS
SM
BASE
MD
LG
XL
XXL

`${FONT_SIZE.BASE}` is 16px and `${SPACING.BASE}` is 20px

if its too big, go down to SM or XS. if its too little go up to MD, LG, XL

The same works concept also works for your shades of gray, just slightly different language:

```
  WHITE: ‘#fff’,
  GREY_LIGHTEST: ‘#F7F7F7’,
  GREY_LIGHTER: ‘#E5ECF3’,
  GREY_LIGHT: ‘#D8D8D8’,
  GREY: ‘#B7BFC6’,
  GREY_DARK: ‘#7B8695’,
  GREY_DARKER: ‘#7B8695’,
  GREY_DARKEST: ‘#282D32’,
  BLACK: ‘#000’,
```

So, you can just put gray grey and then go darker or lighter if its not right.

Breakpoints haven’t been updated yet (BREAK_POINTS.SCREEN_XS) but that’s just a tedious thing if you guys like this system.

### Sreenshots!

<img width="355" alt=" 2020-02-06 at 11 41 01 PM" src="https://user-images.githubusercontent.com/7924827/74012131-60429e00-493e-11ea-8f62-522539cc5d19.png">
<img width="1232" alt=" 2020-02-06 at 11 34 36 PM" src="https://user-images.githubusercontent.com/7924827/74012133-633d8e80-493e-11ea-94bb-84f4aa29fb69.png">
<img width="1249" alt=" 2020-02-06 at 11 34 15 PM" src="https://user-images.githubusercontent.com/7924827/74012137-646ebb80-493e-11ea-98bb-40591cae2f09.png">
<img width="1253" alt=" 2020-02-06 at 11 34 07 PM" src="https://user-images.githubusercontent.com/7924827/74012138-65075200-493e-11ea-8491-4375917c9e7c.png">
<img width="415" alt=" 2020-02-06 at 11 33 52 PM" src="https://user-images.githubusercontent.com/7924827/74012143-66387f00-493e-11ea-9499-4a09833f5e21.png">
<img width="408" alt=" 2020-02-06 at 11 33 47 PM" src="https://user-images.githubusercontent.com/7924827/74012144-66d11580-493e-11ea-9a0c-0f5574118a26.png">
<img width="405" alt=" 2020-02-06 at 11 33 43 PM" src="https://user-images.githubusercontent.com/7924827/74012147-66d11580-493e-11ea-9b7a-311e0a83db06.png">
<img width="413" alt=" 2020-02-06 at 11 33 35 PM" src="https://user-images.githubusercontent.com/7924827/74012148-66d11580-493e-11ea-9216-a6a735a4b221.png">
<img width="414" alt=" 2020-02-06 at 11 33 25 PM" src="https://user-images.githubusercontent.com/7924827/74012150-6769ac00-493e-11ea-8e4f-93b92c9fbcef.png">
<img width="424" alt=" 2020-02-06 at 11 31 56 PM" src="https://user-images.githubusercontent.com/7924827/74012152-6769ac00-493e-11ea-9443-09c24a250478.png">
